src/views/shared/base-layout/base-layout.styles.ts

```diff
@@ -0,0 +1,17 @@
+import { createUseStyles } from "react-jss";
+
+import { Theme } from "src/styles/theme";
+
+const useBaseLayoutStyles = createUseStyles((theme: Theme) => ({
+  spinner: {
```

Comment:
Maybe `spinnerContainer` is better for this class

Reply:
Done!